### DISCUSSION TITLE
The correct HTML label cannot be used

### DISCUSSION THREAD

**Discussion by quirkybird** [remarkjs/remark#1250] [Q&A] [answered]
input：
![image](https://github.com/remarkjs/remark/assets/115051391/e42a5f7b-5aec-40e3-8f14-4a94875bf707)
render：
![image](https://github.com/remarkjs/remark/assets/115051391/98361e52-1e2b-4f21-b116-cba81497d33b)

**Comment by ChristianMurphy** [comment, marked as answer]
@quirkybird I assume you are using the same setup as in https://github.com/orgs/remarkjs/discussions/1249?
With `react-markdown`?

Please read the readme, your question is addressed https://github.com/remarkjs/react-markdown#appendix-a-html-in-markdown

**Reply by quirkybird** [reply]
oh, I should have read more about the documentation，embarrassment
